apps/www/components/Frame/User.js

```diff
@@ -28,7 +28,7 @@ export const SignInLink = ({ isOnMarketingPage }) => {
         <IconAccountBox {...colorScheme.set('fill', 'text')} />
       </span>
       <span
-        {...(isOnMarketingPage ? styles.labelMarketing : styles.labelDefault)}
+        {...styles.label}
```

Comment:
changed it to just one label css because labelMarketing and labelDefault were the exact same css XD